### PR TITLE
Voting Routes [GET]

### PR DESCRIPTION
**Pull request by lgv-0**
# Description

Implements half of the overall voting-route functionality

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [x] Complete, tested, ready to review and merge

### CODE REVIEW

**Review by AndrewMaddocks — APPROVED** [review]

**Review by camjmoore — COMMENTED** [review]
Like it

**Review by LeoSanchez89 — APPROVED** [review]

**Review by camjmoore — APPROVED** [review]

**Comment by lgv-0** [comment]
[Endpoints for independent voting](https://trello.com/c/TApCphGH/15-endpoints-for-independent-voting)

File: src/database/entity/Child.ts [hunk 1]
```diff
@@ -1,11 +1,20 @@
-import { Entity, Column, PrimaryGeneratedColumn, ManyToOne, OneToMany } from 'typeorm';
+import {
+    Entity,
```
Comment (ryan-hamblin) [inline src/database/entity/Child.ts:2]: Was this an auto-format from your linter/formatter? It looks good!

File: src/database/entity/Matches.ts [hunk 2]
```diff
@@ -21,6 +22,9 @@ class Matches {
 
     @Column({ nullable: true })
     week: number;
+
```
Comment (ryan-hamblin) [inline src/database/entity/Matches.ts:25]: I'm not sure we need this extra line here? Consider delete?

**Review by ryan-hamblin — COMMENTED** [review]
Everything here looks really clean. I like the formatting, I like how you are organizing these files. Please remove commented out code and remove the spacing issues that I pointed out and you're good to go!

**Pull request merged**